### ISSUE TITLE
Simplify translation of bible books names

### ISSUE THREAD

**Issue by LAfricain** [postiffm/bibledit-desktop#105]
Currently the list of book displayed in locale language is managed by the language_<language>.xml file. If you don't have a file with your language the books names are not translated. I created an empty file named language_none.xml. And I choose it in the properties, then if I have all the book names translated in the po, the books appear well translated.
I propose to remove this language option in the properties in the project entry, it will be easier for the translator. No need to translate two times the book names.
![language_propreties](https://user-images.githubusercontent.com/19690477/55834081-62b74a00-5b19-11e9-95a6-be3d2b8d1bdb.png)

**Comment by postiffm** [comment]
Good day @LAfricain. I'm not understanding all of this just yet. Question: when you do the .pot file translations, I thought you translated the book names as they exist in src/books.cpp. Perhaps those are not connected to the project properties?

**Comment by LAfricain** [comment]
> when you do the .pot file translations, I thought you translated the book names as they exist in src/books.cpp

Yes, this is OK. But this translation of the po is not enough. the list in the main windows is not translated if you don't add an xml file with the translation.
Look, the picture, I translated all th book's names in the po in Italian, I choose the persian xml file language (more visible ;) ) You can see that only the deuterocanonical are displayed in Italian because there are not in the persian xml translation:

![list_books](https://user-images.githubusercontent.com/19690477/55911795-aaa1a400-5be1-11e9-91f7-91eabcb745c3.png)

My idea is to delete this "option" to choose an xml file in the list, the po translation should be enough. I did a test with an xml empty and the books are in the locale language.
If you do the job, I can add all the current files in a po file with the local. The current list is:
language_djimini.xml
language_english.xml
language_farsi.xml
language_french.xml
language_german_einheitsuebersetzung.xml
language_german_gute_nachricht_2.xml
language_german_gute_nachricht.xml
language_german_luther_2.xml
language_german_luther.xml
language_german_revidierte_elberfelder_2.xml
language_german_revidierte_elberfelder.xml
language_indonesian_tanpa_titik.xml
language_indonesian.xml
language_italian.xml
language_matigsalug.xml
language_ndebele.xml
language_portuguese.xml
language_shona.xml

What I noticed too, it is if I have more than one bible windows opened then the local is good for all the bible even if I choose a file in the xml list.
